website/docs/topics/feature-flags/limit-payloads.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 5. Limit payloads. Feature flag payload should be as small as possible. 
+---
+
+Minimizing the size of feature flag payloads is a critical aspect of maintaining the efficiency and performance of a feature toggling system. The configuration of your feature flags can vary in size depending on the complexity of your targeting rules. For instance, if you have a targeting engine that determines whether a feature flag should be active or inactive based on individual user IDs, you might be tempted to include all these user IDs within the configuration payload. While this approach may work fine for a small user base, it can become unwieldy when dealing with a large number of users.
+
+If you find yourself facing this challenge, your instinct might be to store this extensive user information directly in your Feature Flag Control Service feature flagging system. However, this can also run into scaling problems. A more efficient approach is to categorize these users into logical groupings at a different layer and then use these group identifiers when you evaluate flags within your feature flagging system. For example, you can group users based on their subscription plan or geographical location. Find a suitable parameter for grouping users, and employ those group parameters as targeting rules in your feature flagging solution.
```

Comment:
```suggestion
If you find yourself facing this challenge, your instinct might be to store this extensive user information directly in feature flagging system. However, this can also run into scaling problems. A more efficient approach is to categorize these users into logical groupings at a different layer and then use these group identifiers when you evaluate flags within your feature flagging system. For example, you can group users based on their subscription plan or geographical location. Find a suitable parameter for grouping users, and employ those group parameters as targeting rules in your feature flagging solution.
```